spec/graphql/object_type_spec.rb

```diff
@@ -24,5 +24,12 @@
       assert_equal(GraphQL::TypeKinds::NON_NULL, field.type.kind)
       assert_equal(GraphQL::TypeKinds::SCALAR, field.type.of_type.kind)
     end
+
+    it 'exposes defined field property' do
```

Comment:
I think we don't need this, this behavior is also tested in `field_spec`, right?